Data/uomReferenceData.ttl

```diff
@@ -13914,22 +13913,22 @@ gistd:_UnitOfMeasure_electron_volt_second skos:altLabel "electron volt seconds"
 gistd:_UnitOfMeasure_electron_volt_second skos:closeMatch <http://qudt.org/vocab/unit/EV-SEC> .
 gistd:_UnitOfMeasure_electron_volt_second skos:prefLabel "electron volt second" .
 gistd:_UnitOfMeasure_electron_volt_second skos:scopeNote "1 electron volt second = 0.0000000000000000001602176634 x kilogram meterSquared per second" .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom gist:conversionFactor "0.000000001602176634"^^xsd:decimal .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom gist:isMemberOf gistd:_UnitGroup_force .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom rdf:type gist:UnitOfMeasure .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:altLabel "electronvolts per angstrom" .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:closeMatch <http://qudt.org/vocab/unit/EV-PER-ANGSTROM> .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:definition "from QUDT: unit electronvolt divided by the unit angstrom" .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:prefLabel "electronvolt per angstrom" .
-gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:scopeNote "1 electronvolt per angstrom = 0.000000001602176634 x kilogram meter per secondSquared" .
-gistd:_UnitOfMeasure_electronvolt_per_meter gist:conversionFactor "0.0000000000000000001602176634"^^xsd:decimal .
-gistd:_UnitOfMeasure_electronvolt_per_meter gist:isMemberOf gistd:_UnitGroup_force .
-gistd:_UnitOfMeasure_electronvolt_per_meter rdf:type gist:UnitOfMeasure .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:altLabel "electronvolts per meter" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:closeMatch <http://qudt.org/vocab/unit/EV-PER-M> .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:definition "from QUDT: unit electronvolt divided by the SI base unit metre" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:prefLabel "electronvolt per meter" .
-gistd:_UnitOfMeasure_electronvolt_per_meter skos:scopeNote "1 electronvolt per meter = 0.0000000000000000001602176634 x kilogram meter per secondSquared" .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom gist:conversionFactor "0.000000001602176634"^^xsd:decimal .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom gist:isMemberOf gistd:_UnitGroup_force .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom rdf:type gist:UnitOfMeasure .
+gistd:_UnitOfMeasure_electron_volt_per_angstrom skos:altLabel "electron_volts per angstrom" .
```

Comment:
```suggestion
gistd:_UnitOfMeasure_electronvolt_per_angstrom skos:altLabel "electronvolts per angstrom" .
```

For lines 13916-13931, it looks like "electronvolt" (no space) is the unit name, though "electron volt" is also acceptable. So the "electronvolt" -> "electron_volt" changes can be reverted if we want, OR if we want to keep "electron volt" throughout, the underscore is showing up in the prefLabels and altLabels and needs to be removed.